src/search.ts

```diff
@@ -65,23 +92,28 @@ export class Search {
    * @returns The chainable search object
    */
   kits(): Search {
+    this._context = "global";
     this._queryType = "jump_to";
     return this.ofType("kit");
   }
   /**
    * Restrict results to sections
+   * Only applies to global searches.
    * @returns The chainable search object
    */
   sections(): Search {
+    this._context = "global";
     this._queryType = "jump_to";
     return this.ofType("section");
```

Comment:
I don't love that you can easily call multiple of these context functions on the same search object which could make for a weird query but I called that out in the docs.

Reply:
We can only lead the horse to water 🐴